BepInEx.Preloader.Unity/BepInEx.Preloader.Unity.csproj

```diff
@@ -25,11 +21,5 @@
   <ItemGroup>
     <PackageReference Include="HarmonyX" Version="2.1.1" />
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Properties" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Remove="Properties\AssemblyInfo.cs" />
-  </ItemGroup>
```

Comment:
How is this related to semver change? This should be in a separate PR for better testability and ability to revert it if needed.